docs: Update ko-kr README.md (#4617)
quisquous/cactbot: README.md

![oopsy screenshot](screenshots/promo_oopsy.png)

> You can copy oopsy logs to clipboard by clicking it. (You have to uncheck `Enable Clickthrough` checkbox from the overlay option.)

Reviewer: ```suggestion
You can copy oopsy lines to the clipboard by clicking them.
(You may need to uncheck `Enable Clickthrough` checkbox from the OverlayPlugin option.)
```

Some small edits, and also adding a line break to follow https://sembr.org/.

Author: Oh... I didn't know sembr. I'll review again with that.
